api/tests/unit/domain/usecases/get-user-scorecard_test.js

```diff
@@ -0,0 +1,106 @@
+const { sinon, expect, domainBuilder } = require('../../../test-helper');
+const { UserNotAuthorizedToAccessEntity } = require('../../../../lib/domain/errors');
+const getUserScorecard = require('../../../../lib/domain/usecases/get-user-scorecards');
+
+describe('Unit | UseCase | get-user-scorecard', () => {
+
+  let competenceRepository;
+  let smartPlacementKnowledgeElementRepository;
+
+  beforeEach(() => {
+    competenceRepository = { list: sinon.stub() };
+    smartPlacementKnowledgeElementRepository = { findUniqByUserId: sinon.stub() };
+  });
+
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  context('Access management', () => {
```

Comment:
le nom de ces contextes n'est pas très parlant

Reply:
Je pense que Access management a été choisi car on check si on peut resolve et si on peut reject. Qu'est-ce qui te conviendrait comme nom ?

Reply:
un context rajoute un élément de contexte sur les tests à réaliser. Par exemple `context('when user is connected')` et plus loin `context('when user is disconnected')`

Reply:
J'avais fait la remarque sur le pixScore, DARI avait pris en compte ma remarque, tu peux regarder ce qu'il a changé :)

Reply:
✅